examples/python/keras/func_cifar10_alexnet.py

```diff
@@ -40,7 +40,7 @@ def top_level_task():
     image = x_train[i, :, :, :]
     image = image.transpose(1, 2, 0)
     pil_image = Image.fromarray(image)
-    pil_image = pil_image.resize((229,229), Image.NEAREST)
+    pil_image = pil_image.resize((229,229), Image.Resampling.NEAREST)
```

Comment:
What is this change for?

Reply:
They belong to #423.

Reply:
This is to resolve a warning from the pil_image python library